### DISCUSSION TITLE
Get the hash of the site title and icon at the same time

### DISCUSSION THREAD

**Discussion by x-Ai** [projectdiscovery/httpx#660] [Q&A] [answered]
### Please describe your feature request:

Currently, when the **-title** and **-favicon** parameters are used together, the **-title** parameter only detects the title of the **http(s)://host/favicon.ico** icon page, but not the **http(s)://host** page title. I recommend getting both the page title and also the favicon.ico hash when both parameters are enabled. Thank you very much！
```
./httpx -l host.txt -sc -cl -ct -title -favicon -stats
```
### Describe the use case of this feature:

![image](https://user-images.githubusercontent.com/5061489/171902449-ca9e921b-74f4-4dcd-a238-e0a2e50e92f7.png)

**Comment by ehsandeep** [comment]
@x-Ai What you suggested will require 2 requests, i.e `/` and `/favicon.ico`, `-favicon` is a unique flag for specific use cases and can not be merged with regular / default probes, instead you can make use of `-path` flag to to request both paths and collect the same information.

```console
echo hackerone.com | httpx -path "/,/favicon.ico" -title -sc -hash mmh3

    __    __  __       _  __
   / /_  / /_/ /_____ | |/ /
  / __ \/ __/ __/ __ \|   /
 / / / / /_/ /_/ /_/ /   |
/_/ /_/\__/\__/ .___/_/|_|
             /_/              v1.2.1

		projectdiscovery.io

Use with caution. You are responsible for your actions.
Developers assume no liability and are not responsible for any misuse or damage.
https://hackerone.com/favicon.ico [200] [] [595148549]
https://hackerone.com [302] [] [-728214068]
```

**Comment by ehsandeep** [comment, marked as answer]
@x-Ai this will be improved further as suggested - https://github.com/projectdiscovery/httpx/issues/702

**Reply by x-Ai** [reply]
thank you very much for taking my opinion. 👍👍👍
